addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -154,6 +164,16 @@ $navbar-padding-y: if(index(('fill', 'pills', 'outline'), o-website-value('heade
 $navbar-nav-link-padding-x: if(index(('outline', 'block'), o-website-value('header-links-style')), .5rem * 3, null) !default;
 $navbar-nav-link-padding-x: if(o-website-value('header-links-style') == 'border-bottom', .5rem * 2, null) !default;
 
+// Accordion
+
+$accordion-border-width: 0 !default;
+$accordion-button-color: inherit !default;
+$accordion-button-active-bg: inherit !default;
+$accordion-button-active-color:  inherit !default;
+$accordion-icon-color: $accordion-button-color !default;
+$accordion-icon-width: 1rem !default;
+$accordion-icon-transform: rotate(0deg) !default;
+$accordion-button-icon: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' fill='#{$accordion-icon-color}'><path transform='#{rotate(90, 8, 8)}' fill-rule='evenodd' d='M1.646 4.646a.5.5 0 0 1 .708 0L8 10.293l5.646-5.647a.5.5 0 0 1 .708.708l-6 6a.5.5 0 0 1-.708 0l-6-6a.5.5 0 0 1 0-.708z'/></svg>") !default;
```

Comment:
Same for these ones: if this does not depend on website values (website user custo / theme), then it should be in web/web_editor